JSON_for_IO-Link.yaml

```diff
@@ -3,12 +3,24 @@ info:
   version: 1.1.0
   title: JSON for IO-Link Specification
   description: >-
-    This is an openapi specification for IO-Link gateways, masters and devices. You can find out more about IO-Link
-    at [http://www.io-link.com](http://www.io-link.com)
 
     # Description
-
+    
+    This is an openapi specification for IO-Link gateways, masters and devices. You can find out more about IO-Link
+    at [http://www.io-link.com](http://www.io-link.com)
+    
     * Draft for version 1.1.0
+    
+    # Specification References:
+    
+    >  [JSON Integration for IO-Link - Draft V2.0](http://www.io-link.com/io-link/10.222/V2.0)
```

Comment:
The following links are not working.

Reply:
These are URIs (Identifiers) in the first place. Maybe one day the IO-Link Consortium will make them work as URL links. i.e. to access the standards in an online fashion.

Reply:
If this is confusing someone, we could write URI in front of them. But I would leave  it as it is and rather ask the IO-Link-consortium that their webmaster redirects them to the current download page.

Reply:
Do you know the person who is responsible for doing that and will you arrange this?

Reply:
Frank Moritz ?
https://www.io-link-projects.com/view_crs_page.php?project_id=119
But we should not block ourselves because with that!